team-bimmer/ribbit
# Add Static HTML rendering

**WHAT?**
Compiled users applications client side files and copied to ribbits folder.
Send requests to each route that user specifies in ribbit.routes
save html files to user folder in ribbit.statics directory
1. We use fs-extra to create the directories necessary to keep a manageable file structure when writing html files to user directory
2. rimRaf was used to delete the compiled files in ribbits folder once ribbit build finishes its entire process
3. webpack is used to compile and save files to ribbits folder in correspondence  to the routes provided for each component in ribbit.routes

**WHY?**
Automatically generate static files for each route in the users client side code
**Testing Steps**
Pull this branch into your local machine
Run `ribbit `build` in demo application
Check ribbit.statics folder
